nixos/modules/services/misc/ombi.nix

```diff
@@ -59,7 +65,7 @@ in {
         Type = "simple";
         User = cfg.user;
         Group = cfg.group;
-        ExecStart = "${pkgs.ombi}/bin/Ombi --storage '${cfg.dataDir}' --host 'http://*:${toString cfg.port}'";
+        ExecStart = "${pkgs.ombi}/bin/Ombi --storage '${cfg.dataDir}' --host 'http://*:${toString cfg.port}' --baseurl ${cfg.baseUrl}";
```

Comment:
`${lib.getExe pkgs.ombi}` would be nice.

Reply:
Done in [883d86f](https://github.com/NixOS/nixpkgs/pull/284311/commits/883d86fefaa8833e220bb0a9ebc4a41e05c79c26)